src/printer.js

```diff
@@ -1,5 +1,14 @@
 "use strict";
 
+// other cases that are unstable/problematic:
```

Comment:
Yeah that was just a note-to-self since I was coding without internet. I'll be back online in an hour or two and will create issues if you don't beat me to it 😉